docs/source/index.rst

```diff
@@ -0,0 +1,20 @@
+.. CPP_PTB documentation master file, created by
+   sphinx-quickstart on Sun Oct 25 14:42:27 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to CPP_PTB's documentation!
+===================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+
```

Comment:
```suggestion
    function_description
```

Adding this here will add an entry to the tree of the table of content and create a link to the `function_descrption.rst` file.